Mathlib/Algebra/Pointwise/Stabilizer.lean

```diff
@@ -57,8 +59,72 @@ lemma stabilizer_mul_self (s : Set G) : (stabilizer G s : Set G) * s = s := by
   rw [← mem_stabilizer_iff.1 ha]
   exact smul_mem_smul_set hb
 
+@[to_additive]
+lemma stabilizer_inf_stabilizer_le_stabilizer_union :
+    stabilizer G s ⊓ stabilizer G t ≤ stabilizer G (s ∪ t) := by
+  aesop (add simp [SetLike.le_def, smul_set_union])
+
+@[to_additive]
+lemma stabilizer_inf_stabilizer_le_stabilizer_inter :
+    stabilizer G s ⊓ stabilizer G t ≤ stabilizer G (s ∩ t) := by
+  aesop (add simp [SetLike.le_def, smul_set_inter])
+
+@[to_additive]
+lemma stabilizer_inf_stabilizer_le_stabilizer_sdiff :
+    stabilizer G s ⊓ stabilizer G t ≤ stabilizer G (s \ t) := by
+  aesop (add simp [SetLike.le_def, smul_set_sdiff])
```

Comment:
Hmm, these three lemmas make it seems like there's some common generalisation here... If nothing else, can we derive one from the other two by saying something about compls?

Reply:
The generalisation is that `stabilizer G s ⊓ stabilizer G t ≤ stabilizer G (f s t)` as long as `∀ a, a • f s t = f (a • s) (a • t)`

Reply:
oh neat, that's like a binary mul action hom